pages/previewModel.js

```diff
@@ -69,12 +69,41 @@ function setCamera(scene, model) {
     scene.camera.lookAt(center, new Cesium.HeadingPitchRange(heading, pitch, range));
 }
 
-function loadModel(gltf, resetCamera) {
+function loadModelFromContent(gltfContent, gltfRootPath, resetCamera) {
     scene.primitives.removeAll();
+
     var model = scene.primitives.add(new Cesium.Model({
-        gltf: gltf
+        gltf: gltfContent,
+
+        // In theory, by specifying basePath, we should be able to have relative paths
+        // within the glTFContent.  In practice, specifying basePath makes no difference.
+        // This is likely a bug in Cesium (tracked by https://github.com/AnalyticalGraphicsInc/cesium/issues/5319).
+        // For now, we'll continue with specifying the basePath here, but we'll also
+        // make sure that all paths in glTFContent are absolute to work around the bug.
+        // That logic happens at the time when the content gets written to the script
+        // tag in the HTML.
+        basePath: gltfRootPath.replace(/\\/g, "\\\\") + "\\"
```

Comment:
Note that the path changes a little.  The hostname is blank (hence three slashes, not two), and all of the path separators become forward slashes on all platforms.

So for example: 

`C:\path\to\file.txt`

becomes

`file:///C:/path/to/file.txt`